mapie/utils.py

```diff
@@ -635,7 +635,7 @@ def _check_alpha_and_n_samples(
     else:
         alphas_ = alphas
     for alpha in alphas_:
-        if n < np.max([1/alpha, 1/(1-alpha)]):
+        if alpha == 0 or alpha == 1 or n < np.max([1/alpha, 1/(1-alpha)]):
```

Comment:
To me that change is not trivial:

- Maybe checking for alpha == 1 or 0 is done (or should be done) earlier in the code
- If we decide to check this, we need to update the docstring and the error message as well